Tools/TestWebKitAPI/Tests/WebKitCocoa/AdaptiveImageGlyph.mm

```diff
@@ -394,6 +394,68 @@ - (NSAttributedString *)contentsAsAttributedString
     EXPECT_FALSE(removedPNGAttachmentInfo.shouldPreserveFidelity);
 }
 
+TEST(AdaptiveImageGlyph, InsertWKAttachmentsOnPaste)
+{
+    RetainPtr adaptiveImageGlyphData = [NSData dataWithContentsOfURL:[NSBundle.mainBundle URLForResource:@"adaptive-image-glyph" withExtension:@"heic" subdirectory:@"TestWebKitAPI.resources"]];
+
+    RetainPtr adaptiveImageGlyph = adoptNS([[NSAdaptiveImageGlyph alloc] initWithImageContent:adaptiveImageGlyphData.get()]);
+
+#if PLATFORM(MAC)
+    RetainPtr font = [NSFont systemFontOfSize:16];
+#else
+    RetainPtr font = [UIFont systemFontOfSize:16];
+#endif
```

Comment:
We could save some typing here: `RetainPtr font = [WebCore::CocoaFont systemFontOfSize:16];`